recipes/recipes_emscripten/r-base/recipe.yaml

```diff
@@ -10,21 +10,21 @@ source:
   url: https://cran.r-project.org/src/base/R-4/R-${{ version }}.tar.gz
   sha256: 1578cd603e8d866b58743e49d8bf99c569e81079b6a60cf33cdf7bdffeb817ec
   patches:
-  - patches/0001-Fix-bad-substitution-error.patch
-  - patches/0002-Disable-libcurl.patch
-  - patches/0003-Remove-unsupported-sigsuspend.patch
-  - patches/0004-Disable-internet-module.patch
-  - patches/0005-Use-R-binary-from-build-prefix.patch
-  - patches/0006-Ignore-crude-heuristic-file-size.patch
-  - patches/0007-Build-docs-with-Rscript-binary.patch
-  - patches/0008-Set-default-R_Home.patch
-  - patches/0009-Use-basenames-for-compilers.patch
-  - patches/0010-Fix-iconv-checks-when-using-internal-lapack-blas.patch
-  - patches/0011-Build-with-fPIC-only-modules-that-need-it.patch
-  - patches/0012-Build-with-atomics-and-bulkmemory.patch
+    - patches/0001-Add-emscripten-platform-to-configure-script.patch
+    - patches/0002-Disable-libcurl.patch
+    - patches/0003-Disable-internet-module.patch
+    - patches/0004-Fix-iconv-checks-when-using-internal-lapack-blas.patch
+    - patches/0005-Add-sigsuspend-stub-for-Emscripten-compatibility.patch
+    - patches/0006-Build-with-fPIC-the-modules-that-need-it.patch
+    - patches/0007-Disable-which-for-emscripten.patch
+    - patches/0008-Use-linux-Rscript-executable-when-building-the-docs.patch
+    - patches/0009-Use-source-files-directly.patch
+    - patches/0010-Ignore-warnings-when-building-Rdts-files.patch
+    - patches/0011-Use-cross_libraries-script-when-cross-compiling.patch
+    - patches/0012-Install-wasm-files.patch
 
 build:
-  number: 3
+  number: 4
```

Comment:
[Bot updated version](https://github.com/emscripten-forge/recipes/pull/1429) but did not reset build number :face_with_diagonal_mouth:

Reply:
@DerThorsten :)